core/iwasm/fast-jit/fe/jit_emit_memory.c

```diff
@@ -389,6 +389,10 @@ jit_compile_op_i64_store(JitCompContext *cc, uint32 align, uint32 offset,
         goto fail;
     }
 
+    if (jit_reg_is_const(value) && bytes < 8) {
```

Comment:
what does this for?
if prefer to let the compiler do the job about const values, should skip the following `switch`, right?